### PR TITLE
SL-677: Add Inpatient Ward and Queue list Apps on the home page

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
             patientVisitsPageUrl = "/coreapps/patientdashboard/patientDashboard.page?patientId={{patient.patientId}}";
             patientVisitsPageWithSpecificVisitUrl = patientVisitsPageUrl + "&visitId={{visit.visitId}}";
         }
-        patientEncountersPageUrl="/pihcore/visit/visit.page?patient={{patient.uuid}}#/encounterList";
+        patientEncountersPageUrl = "/pihcore/visit/visit.page?patient={{patient.uuid}}#/encounterList";
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:213]: a bunch of auto-formatting stuff which can be ignored

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 2]
```diff
@@ -600,23 +604,32 @@ private void enableMCHTriage(Config config) {
                     and(sessionLocationHasTag("MCH Triage Location"),
                             and(patientIsFemale(), patientIsAdult()),
                             visitDoesNotHaveEncounterOfType(SierraLeoneConfigConstants.ENCOUNTERTYPE_SIERRALEONEMCHTRIAGE_UUID))));
+
+            apps.add(addToHomePage(app(CustomAppLoaderConstants.Apps.MCH_TRIAGE_QUEUE,
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:608]: I figured I'd bundle the triage queue app within the "MCH Triage" component.

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 3]
```diff
@@ -914,6 +928,29 @@ private void enableADT() {
                 findExtensionById(CustomAppLoaderConstants.EncounterTemplates.NO_DETAILS), "fas fa-fw fa-sign-out-alt", null, true, null, null);
     }
 
+    private void enableADTO3() {
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:931]: certainly open to a better name than "ADT O3", but nothing came to mind.

Comment (cioan) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:931]: maybe not using numbers in the name of the method?

**Review by cioan — APPROVED** [review]
LGTM, thanks @mogoodrich !

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 4]
```diff
@@ -770,25 +783,25 @@ private void enableVitals() {
     }
 
     private void enableConsult() {
-        if (config.getCountry().equals(ConfigDescriptor.Country.PERU) ){
+        if (config.getCountry().equals(ConfigDescriptor.Country.PERU)) {
```
Comment (lnball) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:786]: Do we want to remove Peru?

**Review by lnball — APPROVED** [review]
LGTM

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java (hunk 4 above)
Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:786]: This was just a stuff the auto-formatter fixed... but, yeah, we should clean up up the Peru stuff... feel free to ticket? :)

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java (hunk 3 above)
Comment (mseaton) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:931]: Is there an "ADT non-O3"?  If not, why not just ADT?

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 5]
```diff
@@ -914,6 +928,18 @@ private void enableADT() {
...
 
+    private void enableADTO3() {
+
+        apps.add(addToHomePage(app(CustomAppLoaderConstants.Apps.INPATIENT_WARD,
+                        CustomAppLoaderConstants.Apps.INPATIENT_WARD,
+                        "fas fa-fw fa-hospital",
+                        "spa/ward",
+                        "App: emr.inpatients",
+                        null),
+                sessionLocationHasTag("Inpatients App Location")));
```
Comment (mseaton) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:939]: What is this tag?  Shouldn't this be "Admission Location"?

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java (hunk 3 above)
Comment (mseaton) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:931]: NM.  I see below.  I'm not super concerned about the naming.

**Review by mseaton — APPROVED** [review]
LGTM, aside from the question about location tag name...

**Pull request merged**